### ISSUE TITLE
Raw logits

### ISSUE THREAD

**Issue by DeodatoBastos** [soda-inria/tabicl#25]
Hello! First, I would like to thank you for sharing this amazing work!

My question is about the raw logits. I analyzed the shared code and wasn't able to find a way to obtain the raw logits instead of the probabilities.

I would be happy to contribute to the project by writing a function that returns only the logits.

Please let me know if I'm misunderstanding something or if you would like my help and if it aligns with the project's goals.

Thanks again!

**Comment by dholzmueller** [comment]
Hi, you can always reconstruct the logits up to a shift by computing `np.log(probas)`. Are you interested in the shift specifically, even though it doesn't affect the probabilities?

**Comment by DeodatoBastos** [comment]
Thank you for getting back to me. I will try to continue with that.
